Foundation-Mission-Structure-and-Strategy-Proposal.md

```diff
@@ -0,0 +1,76 @@
+# Node.js Foundation Mission, Structure and Strategy
+
+_Proposal to the Node.js Foundation TSC and Board, by Rod Vagg_
+
+**Goal #1:** Agree on a basic statement of mission for the Foundation, upon which all execution, by each group within the Foundation, can be tested and justified.
+
+**Goal #2:** Agree on a strategy generation and execution structure, and the relationship between the main bodies within that structure, that can best achieve the mission of the Foundation.
+
+## Foundation Mission Statement
+
+**The Node.js Foundation exists to increase the adoption of Node.js.**
+
+This goal is achieved by:
+
+1. Maintaining the health of the core Node.js project
+2. Fostering and encouraging a healthy Node.js open source ecosystem
+3. Fostering and encouraging a healthy commercial ecosystem of training, products and services for Node.js
+
+_This is an executable form of the Node.js Foundation Purpose as presented in its by-laws, Section 1.4_
+
+### Fundamendals
+
+The proposed mission statement stands on a fundamental raison d’être: “increase the adoption of Node.js”
```

Comment:
don't you come from semi-French land?

> "the most important reason or purpose for someone or something's existence."

I'll change it, I hate pronouncing this when speaking and I'll need to say this to the Board.